package.json

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "devDependencies": {
     "@commitlint/config-conventional": "^17.1.0",
+    "@salesforce/core": "^3.36.2",
```

Comment:
I was discussing this PR with @RitamAgrawal and she brought up that one of the original goals and design requirements of the E2E tests, was to:
1. only interact with VS Code like a user would
2. not call any of the internal libraries we use

I see arguments for and against using core here...
For:
The places where salesforce/core are used is when we are automating org auth calls to sfdx, so this isn't UI or user actions that we're automating

Against:
By adding core, this opens the door for APIs to be called instead of controlling VS Code like a user would.

I only am here for a few more days, so I think you & @CristiCanizales  and @CristiCanizales  should discus this.